src/base/utils/misc.cpp

```diff
@@ -237,52 +237,52 @@ QPoint Utils::Misc::screenCenter(const QWidget *w)
 }
 #endif
 
-QString Utils::Misc::unitString(Utils::Misc::SizeUnit unit)
+QString Utils::Misc::unitString(Utils::Misc::SizeUnit unit, bool isSpeed)
 {
-    return QCoreApplication::translate("misc",
-                                       units[static_cast<int>(unit)].source, units[static_cast<int>(unit)].comment);
+    const auto &unitString = units[static_cast<int>(unit)];
+    QString ret = QCoreApplication::translate("misc", unitString.source, unitString.comment);
+    if (isSpeed)
+        ret += QCoreApplication::translate("misc", "/s", "per second");
+    return ret;
 }
-
-// return best userfriendly storage unit (B, KiB, MiB, GiB, TiB, ...)
-// use Binary prefix standards from IEC 60027-2
-// see http://en.wikipedia.org/wiki/Kilobyte
-// value must be given in bytes
-// to send numbers instead of strings with suffixes
-bool Utils::Misc::friendlyUnit(qint64 sizeInBytes, qreal &val, Utils::Misc::SizeUnit &unit)
+namespace
```

Comment:
shouldn't we have namespace declaration at top of the file?

Reply:
Thanks! [;)](https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#11-not-covered-above)